easybuild/easyconfigs/b/beagle-lib/beagle-lib-2_1_2-foss-2016a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '2_1_2'
```

Comment:
this is a very unnatural looking version number...

please use `version = '2.1.2'`, and derive the version to be used in the source filename below